minbpe/base.py

```
@@ -6,21 +6,26 @@
+from itertools import pairwise
 from collections import Counter
+import unicodedata
+
 
 # -----------------------------------------------------------------------------
 # a few helper functions useful for both BasicTokenizer and RegexTokenizer
 
-def get_stats(ids:list[int], counts:Counter[tuple[int, int]]=None):
+
+def get_stats(
+    ids: list[int], counts: Counter[tuple[int, int]] = None
```

Ruff (0.14.4)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)